app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -30,6 +35,12 @@ class StorySummaryViewModel(
   fun setStoryPercentage(storyPercentage: Int) {
     this.storyPercentage.set(storyPercentage)
     storyProgressPercentageText.set(computeStoryProgressPercentageText(storyPercentage))
+    contentDescriptionText.set(computeContentDescriptionText())
+  }
+
+  fun setChaptersInProgress(chaptersInProgress: Int) {
```

Comment:
Could this & the above maybe be set at the same time? Otherwise there's the chance of the content description only being partly correct which is a bit awkward.